devfile.yaml

```diff
@@ -1,57 +1,21 @@
 apiVersion: 1.0.0
 metadata:
   name: che-server-in-che
-projects:
-  - name: che
-    source:
-      type: git
-      location: 'https://github.com/eclipse/che.git'
```

Comment:
You mentioned in the PR description that the project is removed to be able to use forks. It is nice idea, but people who just want to try to play with / build Che might be slightly disappointed with absence of the project itself. Maybe it is worth at least add a clone command?

Reply:
I don't know what's the best practice here. I can add command but it feels clumsy and it would pollute the workspace.

Reply:
>You mentioned in the PR description that the project is removed to be able to use forks. It is nice idea, but people who just want to try to play with / build Che might be slightly disappointed with absence of the project itself.

The project would be there. It will be taken from the context. You don't oblige to specify it.  see more https://www.eclipse.org/che/docs/che-7/making-a-workspace-portable-using-a-devfile/#preparing-a-minimal-devfile_making-a-workspace-portable-using-a-devfile

Reply:
the profit of this change is to make devfile `fork-friendly`.

Reply:
I see, it will clone the repository in which the devfile is located.